Sources/SwiftRedis/RedisResp.swift

```diff
@@ -22,10 +22,10 @@ enum RedisRespStatus {
     case connected
 }
 
-class RedisResp {
+public class RedisResp {
```

Comment:
Why do you need to change the visibility?
That class is public, but no methods are.
Could you elaborate on the use case please?

Reply:
To gain access to socket - so a user can change `readBufferSize` if needed, or disconnect, or exhibit some sort of manual control if needed.